common/rabbitmq/templates/etc/_rabbitmq-custom-config.tpl

```diff
@@ -0,0 +1,15 @@
+{{ if .Values.customConfig }}
+## RabbitMQ custom config
+## Generated by helm template etc/_rabbitmq-custom-config.tpl
+## For more information please find helm chart
+##     https://github.com/sapcc/helm-charts/tree/master/common/rabbitmq
+
+## For more information about RabbitMQ configuration please find
+##     https://www.rabbitmq.com/configure.html
+
+{{- range $k, $v := .Values.customConfig }}
+{{ if not (eq $v nil) }}
```

Comment:
I understand to test for that is [described here](https://github.com/Masterminds/sprig/issues/53#issuecomment-483414063)
```gotpl
{{ if kindIs "invalid" $value }}
```